src/cmds/pbs_rstat.c

```diff
@@ -222,11 +277,81 @@ display(struct batch_status *resv, int how)
 	}
 
 	while (cur != NULL) {
-		display_single_reservation(cur, how);
+		display_single_reservation(cur, how, json_out);
 		cur = cur->next;
 	}
 }
 
+/**
+ * @brief
+ *	Encodes the information in batch_status structure to json format
+ *
+ * @param[in] *bstat - structure containing node information
+ *
+ * @return - Error code
+ * @retval   1 - Failure
+ * @retval   0 - Success
+ *
+ */
+static int
+encode_to_json(struct batch_status *bstat)
```

Comment:
Check to make sure bstat is not NULL since it is referenced later as bstat->name, etc...

Reply:
This is a PR from a summer student who has left as classes resumed.  I should be able to take this over.  Do you have a suggestion as to the best way to proceed with the PR?